### PR TITLE
Update Landing page intro

### PR DESCRIPTION
**Pull request by taysea**
Preview: https://deploy-preview-572--keen-mayer-a86c8b.netlify.com/

Updates the top section of the landing page. Currently the same svg is being used for dark and light mode because Chris has not provided a dark mode version yet.

### CODE REVIEW

File: aries-site/src/pages/index.js [hunk 1]
```diff
@@ -57,10 +57,10 @@ const Index = () => {
       <Meta title={title} description={pageDetails.seoDescription} />
       <Box gap="large">
         <HomeTiles>
```
Comment (halocline) [inline aries-site/src/pages/index.js]: Was wondering if we should change this to be a `row-responsive Box` or a `Grid` with flexible rows? That would prevent this huge gap in single column mode: 
Screen Shot 2020-03-25 at 1 34 30 PM

Comment (britt6612) [reply aries-site/src/pages/index.js]: This problem is also in the current site Correct .?

**Comment by halocline** [comment]
correct.

File: aries-site/src/pages/index.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/pages/index.js]: It is just due to the nature of the layout being created with Tiles that have consistent height, so the text content doesn't fill the height fully in this width -- Let me play with an idea I have and push a commit up

Comment (taysea) [reply aries-site/src/pages/index.js]: Think I fixed it!

Comment (halocline) [reply aries-site/src/pages/index.js]: Sweet! Thank you. I will preview as soon as the build completes.

**Comment by britt6612** [comment]
Spacing looks much better!

**Pull request merged**